packages/modelviewer.dev/src/docs-and-examples/utils.ts

```diff
@@ -103,5 +104,33 @@ export function init(docsOrExamples: string) {
   });
 }
 
+export function toggleSidebar() {
+  const root = document.documentElement;
+  const nav = document.getElementById('sidenav')!;
+  const zero = 0;
+  const stringZero = zero.toString().concat('px');
+  const threeHundred = 300;
+  const stringThreeHundred = threeHundred.toString().concat('px');
+
+  if (nav.offsetWidth > 150) {
+    root.style.setProperty('--sidebar-width', stringZero);
+  } else {
+    root.style.setProperty('--sidebar-width', stringThreeHundred);
+  }
+}
+
+(self as any).toggleSidebar = toggleSidebar;
```

Comment:
Are these required? I'm trying to remember what this does...

Reply:
it seems to be required for the to call within the html code